libraries/bsg_manycore_config.h

```diff
@@ -501,9 +501,9 @@ extern "C" {
         }
 
         /**
-         * Return the number of DRAM channels in the system.
+         * ???
          * @param[in] cfg A configuration initialized from the manycore ROM.
-         * @return 1 if cache is a supported memory system feature, 0 otherwise.
+         * @return ???
```

Comment:
@mrutt92 What should I put here for comments?

Reply:
Something that explains what this function does.

Reply:
I don't *know* what it does

Reply:
How about it returns an identifier for the memory system type?

Reply:
Thanks!